### ISSUE TITLE
Implement cluster info window / custom look

### ISSUE THREAD

**Issue by DDRBoxman** [DDRBoxman/google-maps-ios-utils#8]
googlemaps/google-maps-ios-utils#4

**Comment by kyramckenna** [comment]
Yes Ive got the same issue. The GMSMarker info contained in the Spot object is lost when its added to the cluster

**Comment by tiagoMaua** [comment]
I have same problem... 

When i try show infowindow, i be using the method:

- (BOOL)mapView:(GMSMapView *)mapView didTapMarker:(GMSMarker *)marker {
    _mapView.selectedMarker = marker;
    return YES;
}

 But this not show nothing.

 In debug i can add the title label and snippet, like imagem below:
![captura de tela 2015-09-02 as 02 23 44](https://cloud.githubusercontent.com/assets/3158966/9623660/57b2233a-511a-11e5-9a2a-f4f446a02c42.png)


But when method mapView:didTapMarker is called, this receive a marker only position, without the title and snippet, like image below:

![captura de tela 2015-09-02 as 02 25 39](https://cloud.githubusercontent.com/assets/3158966/9623668/81012c68-511a-11e5-81c2-6f578946d702.png)

How can show Infowindow?

**Comment by mumensh** [comment]
Go to GDefaultClusterRenderer.m file, in clustersChanged method, add the following:
```
marker.title = cluster.marker.title;
marker.snippet = cluster.marker.snippet;
```
then the title and snippet will show normally.
